microsoftgraph/microsoft-graph-explorer-v2
# disables run query button when query is running

## Overview

disables run query button when query is running

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Click run query button
* It should disable and show a loading spinner until the query is complete
